gradlew

```diff
@@ -1,164 +1,164 @@
-#!/usr/bin/env bash

```

Comment:
This will be probably the only file that may change like this and is generated. Removing would mean that the user would need to have Gradle installed on its system. It does not change often, it's quite rare to be honest.